.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    },
+  },
+  "rules" : {
+    "semi": [2, "always"],
+    "strict": [2, "global"]
+  },
+  "extends": [
+    // "./node_modules/mofo-style/.eslintrc.yaml"
```

Comment:
.eslintrc is technically not json (that would be `eslintrc.json`). The .eslintrc file is deprecated but given that we want to move the code to mofo-style, as a temporary file this seemed fine.